examples/timeseries/timeseries_weather_forecasting.py

```diff
@@ -0,0 +1,418 @@
+"""
+Title: Timeseries forecasting for weather prediction
+
+**Authors:**
+
+ - [Prabhanshu Attri](https://prabhanshu.com/github),
+ - [Yashika Sharma](https://github.com/yashika51),
+ - [Kristi Takach](https://github.com/ktakattack),
+ - [Falak Shah](https://github.com/falaktheoptimist)
+<br>
+
+**Date created:** 2020/06/23  <br>
```

Comment:
Not <br> tags in this section. Use a standard header, e.g. https://github.com/keras-team/keras-io/blob/master/examples/generative/deep_dream.py#L1-L6